Subject: Remove hacky code
From: Evangelos Ribeiro Tzaras <devrtz@fortysixandtwo.eu>

* src/calls-call-window.c: Put ugly code behind a define
--- a/src/calls-call-display.c
+++ b/src/calls-call-display.c
@@ -519,7 +519,9 @@
 
   call_state_changed_cb (self, calls_call_get_state (self->call));
 
+#ifdef CALLS_USE_UGLY_CODE
   ugly_hacks (self);
+#endif
 
   G_OBJECT_CLASS (calls_call_display_parent_class)->constructed (object);
 }
